easybuild/easyconfigs/c/cutadapt/cutadapt-2.1-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics (SIB)
+# Biozentrum - University of Basel
+# Modified by: Adam Huffman, Jonas Demeulemeester
+# The Francis Crick Institute
+# Modufied by: Albert Bogdanowicz
+# Institute of Biochemistry and Biophysics PAS
+
+easyblock = 'PythonPackage'
+
+name = 'cutadapt'
+version = '2.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://opensource.scilifelab.se/projects/cutadapt/'
+description = """Cutadapt finds and removes adapter sequences, primers, poly-A tails and
+ other types of unwanted sequence from your high-throughput sequencing reads."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['09fd222a27cc1eeb571633f2bd54442ea9d4ff668ef1f475fd9d5253a7d315ef']
+
+dependencies = [
+    ('Python', '3.6.6'),
+]
+
```

Comment:
@jdemeul the failing test is because `use_pip` and `download_dep_fail` are now mandatory for `PythonPackage`

Reply:
fixed in 5a747c1603